web/src/routing/routes/UsersAdministration/Groups/GroupDetails/GroupDetails.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState, VFC } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+
+import { Box } from "@mui/material";
+
+import { Button } from "@components/Button/Button";
+import { ListElement } from "@components/ListElement/ListElement";
+import { UsersAdministrationLayout } from "@layouts/UsersAdministrationLayout/UsersAdministrationLayout";
+import UsersTable from "../../Users/UsersTable/UsersTable";
+import { EditGroupModal } from "../Modals/EditGroupModal/EditGroupModal";
+import { ConfirmationModal } from "@components/ConfirmationModal/ConfirmationModal";
+import { Paths } from "@routing/paths";
+import { Group } from "@janush-types/group";
+import { User } from "@janush-types/user";
+import { rgbaColors } from "@themes/palette";
+
+const data: User[] = [
+  {
+    id: "1c4a9a8d-b652-421a-b130-9ad680029521",
+    email: "b.szurek@codeandpepper.com",
+    phoneNumber: "+48 666 777 888",
+    access: true,
+    status: true,
+    lastModified: "15-06-2021",
+  },
+  {
+    id: "1c4a9a8d-b652-421a-b130-9ad680029522",
+    email: "b.szurek@codeandpepper.com",
+    phoneNumber: "+48 666 777 888",
+    access: false,
+    status: false,
+    lastModified: "15-06-2021",
+  },
+  {
+    id: "1c4a9a8d-b652-421a-b130-9ad680029523",
+    email: "b.szurek@codeandpepper.com",
+    phoneNumber: "+48 666 777 888",
+    access: true,
+    status: false,
+    lastModified: "15-06-2021",
+  },
+];
+
+interface LocationState {
+  group: Group;
+}
+
+const GroupDetails: VFC = () => {
+  const navigate = useNavigate();
+
+  const location = useLocation();
+  // TODO: change to get group data from backend
+  const { group } = location.state as LocationState;
+
+  const [showEditGroupModal, setShowEditGroupModal] = useState(false);
+  const [showDeletingGroupModal, setShowDeletingGroupModal] = useState(false);
```

Comment:
I see lack of consistency here => `showEdit` vs `showDeleting`. I'd rename `showDeletingGroupModal` to `showDeleteGroupModal` most likely